app/Console/Kernel.php

```diff
@@ -141,6 +141,17 @@ private function scheduleJobsForCollections(Schedule $schedule): void
             ->withoutOverlapping()
             ->dailyAt('5:30');
 
+        $schedule
+            ->command(FetchCollectionNfts::class, [
+                // Limit the number of NFTs to fetch for unsigned wallets, since
+                // it sorts the collections by last_fetched_at eventually it will
+                // fetch all NFTs for all collections
+                '--limit' => config('dashbrd.daily_max_nft_retrieval_for_unsigned_wallets'),
```

Comment:
this limit was intended to be on the collections itself, not their NFTs. Idea is to take a subset of the new collections from unverified users and index those in full and pick up on other ones over the next few days. The issue with limiting it by NFTs is that it may take a very long time to complete it with the currently set limits (e.g. 100 NFTs would require 90 runs to complete for a collection with 9k NFTs)

Reply:
@ItsANameToo it limiting the collections, the setting name was incorrect, fixed on my next batch of changes